drivers/usbdev/rndis.c

```diff
@@ -1384,11 +1389,16 @@ rndis_prepare_response(FAR struct rndis_dev_s *priv, size_t size,
  *
  ****************************************************************************/
 
-static int rndis_send_encapsulated_response(FAR struct rndis_dev_s *priv)
+static int rndis_send_encapsulated_response(FAR struct rndis_dev_s *priv, size_t size)
 {
   FAR struct rndis_notification *notif =
     (FAR struct rndis_notification *)priv->epintin_req->buf;
 
+  /* Mark the response as available in the queue */
+  priv->response_queue_bytes += size;
```

Comment:
Please include an empty line after the comment line, it is part of the Coding Style.

Reply:
Hmm, I already added the empty lines after the style check complained. Are they not visible for you?